docs/releases/1.9.txt

```diff
@@ -360,6 +360,15 @@ Miscellaneous
 * ``CommaSeparatedIntegerField`` validation has been refined to forbid values
   like ``','``, ``',1'``, and ``'1,,2'``.
 
+* ``ModelState`` has been changed to check that the ``to`` and ``through``
```

Comment:
I am not sure these release notes are necessary. At least, I think most users won't know what this means (I don't), and I don't know that we have broken any public APIs?